### PR TITLE
Add Hello World example to intro

### PR DESCRIPTION
**Pull request by marzipankaiser**
To solve https://github.com/effekt-lang/effekt/issues/150

### CODE REVIEW

**Comment by marzipankaiser** [comment]
Adding more content to/rewriting the intro/adding a useful tutorial would probably be good, too.

File: docs/index.md [hunk 1]
```diff
@@ -14,6 +14,13 @@ Effekt language.
   {% endif %}
 {% endfor %}
 
+Let's start with a hello world:
```
Comment (b-studios) [inline docs/index.md:17]:
```suggestion
Let's start with a simple example function that prints `"Hello World!" to the console:
```

**Comment by jiribenes** [comment]
Nitpick: Can the reader test online that the function actually prints `"Hello World!"`?

**Comment by b-studios** [comment]
We can add a 
```effekt:repl
main()
```
below to run it.

**Review by b-studios — APPROVED** [review]

File: docs/index.md [hunk 2]
```diff
@@ -14,6 +14,17 @@ Effekt language.
   {% endif %}
 {% endfor %}
 
+Let's start with a simple example function that prints `"Hello World!" to the console:
```
Comment (jiribenes) [inline docs/index.md:17]: Unbalanced \`:
```suggestion
Let's start with a simple example function that prints `"Hello World!"` to the console:
```

**Pull request merged**